src/components/app-content-switcher/AppContentSwitcher.jsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
+import Switch from '@mui/material/Switch'
+import Stack from '@mui/material/Stack'
+import { defaultStyles } from '~/components/app-content-switcher/AppContentSwitcher.styles'
+
+const AppContentSwitcher = (props) => {
+  const { active, handleChange, tooltipRight, tooltipLeft, spacing, leftText, rightText, colorActive, colorInActive, typographyVariant, styles } = props
+
+  const blocks = [
+    { text: leftText, tooltip: tooltipLeft },
+    { text: rightText, tooltip: tooltipRight },
+  ]
+
+  const renderBlock = ({ text, tooltip }, index) => (
+    <Tooltip arrow key={ index } title={ tooltip }>
+      <Typography color={ index === 0 ? colorActive : colorInActive } data-testid='text' variant={ typographyVariant }>
```

Comment:
why you need this data-testid, you can use getByText

Reply:
refactored